diff --git a/node_modules/@strapi/provider-upload-aws-s3/dist/index.js b/node_modules/@strapi/provider-upload-aws-s3/dist/index.js
index b9fd155..26a531f 100644
--- a/node_modules/@strapi/provider-upload-aws-s3/dist/index.js
+++ b/node_modules/@strapi/provider-upload-aws-s3/dist/index.js
@@ -91,7 +91,6 @@ const index = {
           Bucket: config.params.Bucket,
           Key: fileKey,
           Body: file.stream || Buffer.from(file.buffer, "binary"),
-          ACL: config.params.ACL,
           ContentType: file.mime,
           ...customParams 
         }
@@ -103,6 +102,7 @@ const index = {
         file.url = `https://${upload2.Location}`;
       }
     };
+    
     return {
       isPrivate() {
         return config.params.ACL === "private";
